### ISSUE TITLE
Questions about the code.

### ISSUE THREAD

**Issue by sev777** [Hunter-DDM/knowledge-neurons#5]
Hi,
I find in '3_modify_activation.py', the code 
   `                 _, logits = model(input_ids=input_ids, attention_mask=input_mask, token_type_ids=segment_ids, tgt_pos=tgt_pos, tgt_layer=0, imp_pos=kn_bag, imp_op='remove') `
why the tgt_later is always 0, in kn_bag, some neurons is [9,1000] or [10,1001], not always in layer 0.
And this happens in places like edit and erase.

And in the paper, Figure 4 and Figure 5 is the correct probability change or the probability change about the correct labe, does it mean the remove or enhance can improving probing performance or improving the ranking of the target labe(But the output is wrong) ?

Thanks!

**Comment by Hunter-DDM** [comment]
hi, "tgt_later" is used for getting the FFN weight at a specific layer. In "3_modify_activation.py", we do not need acquire FFN weights, so it can be any value. The layer we aim to modify is stored in "imp_pos=kn_bag".

**Comment by Hunter-DDM** [comment]
It means the removing or enhancing can change the probability of the correct answer. If I understand correctly, that is just what you mean, "improving/decreasing the ranking of the target/correct label (even if the output is still wrong)"
